connectivity/drivers/ble/FEATURE_BLE/TARGET_STM32WB/HCIDriver.cpp

```diff
@@ -509,7 +509,7 @@ class TransportDriver : public CordioHCITransportDriver {
 #if STM32WB55xx
                 switch (p_wireless_info->StackType) {
                     case INFO_STACK_TYPE_BLE_FULL:
-                        if (MBED_ROM_SIZE > 0xCA000)  {
+                        if (MBED_ROM_SIZE > 0xCE000)  {
```

Comment:
Maybe we need to add a line like below (line 517) to check version 1.14 ?

Reply:
Good call, I'll get that added.